src/assets/app.css

```
@@ -88,48 +91,109 @@
 }
 
 @plugin "daisyui/theme" {
-  name: "dark-blue";
+  name: "dark-teal-neutral";
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)